openerp/tools/mail.py

```diff
@@ -123,6 +130,30 @@ def html_sanitize(src, silent=True, strict=False, strip_style=False, strip_class
     return cleaned
 
 
+_style_whitelist = ['width', 'height', 'font-size']
+_style_replace = {
+    ''
+}
+
+def sanitize_attributes(html_root):
```

Comment:
> We could append the code to __call__, maybe.

Yes, add a phase/step to `__call__` to handle attribute transformation as well as the existing removal.